presto-main/src/main/java/io/prestosql/server/protocol/RowIterable.java

```diff
@@ -50,14 +56,18 @@ public RowIterable(ConnectorSession session, List<Type> types, Page page)
     private static class RowIterator
             extends AbstractIterator<List<Object>>
     {
+        private static final ZoneId ZULU = ZoneId.of("Z");
```

Comment:
A particular reason for not using `ZoneOffset.UTC` here?
(we tend to use ZoneOffset.UTC predominantly)

Reply:
No particular reason. They are equivalent, but I'll update it. It saves a lookup.